CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
+* Istio integration is simplified with Cilium build of istioctl. (Backport PR #12356, Upstream PR #10851, @jrajahalme)
+* Istio integration is updated to release 1.5.1, with backported fix for GKE/COS. (Backport PR #12356, Upstream PR #10730, @jrajahalme)
+
+**Bugfixes:**
+* endpoint: Fix data races while accessing GetIdentity() (Backport PR #12021, Upstream PR #11941, @tgraf)
+* Fix bug where etcd session renew would block indefinitely, causing endpoint provision to fail (Backport PR #12341, Upstream PR #12292, @joestringer)
+* Fix bug where identity allocation wouldn't cancel from api timeouts (Backport PR #12352, Upstream PR #12328, @joestringer)
+* helm/operator: fix IPv6 liveness probe address for operator (Backport PR #12341, Upstream PR #12223, @Rolinh)
+* ipcache: Fix deadlock when ipcache GC results in datapath reload (Backport PR #12021, Upstream PR #11950, @tgraf)
+* iptables: Remove '--nowildcard' from socket match (Backport PR #12356, Upstream PR #12248, @jrajahalme)
+* Istio integration has been updated to Istio release 1.5.4 (Backport PR #12356, Upstream PR #11530, @jrajahalme)
+* Istio integration is updated to Istio release 1.5.6. (Backport PR #12356, Upstream PR #12214, @jrajahalme)
+* Istio integration is updated to Istio release 1.5.7. (Backport PR #12356, Upstream PR #12353, @jrajahalme)
+* Stop Cilium from hanging on CNP or CCNP events from Kubernetes if running with 'k8s-event-handover=true' and 'kvstore=""' (Backport PR #12341, Upstream PR #12146, @aanm)
+* The host proxy is updated to Envoy release 1.13.3 (Backport PR #12352, Upstream PR #12343, @jrajahalme)
+
+**CI Changes:**
+* Fix flakey assertion on metrics (Backport PR #12021, Upstream PR #11966, @christarazi)
+* ginkgo-ext: Fix data-race in Writer (Backport PR #12341, Upstream PR #12025, @gandro)
+* test: Add retries to curl command (Backport PR #12356, Upstream PR #11993, @christarazi)
+* test: Download correct cilium-istioctl for the executing OS. (Backport PR #12356, Upstream PR #12109, @jrajahalme)
+* test: Skip Istio test if Ginkgo runs on unsupported runtime. (Backport PR #12356, Upstream PR #11905, @jrajahalme)
+
+**Misc Changes:**
+* k8s: update k8s libraries to v1.16.11 (#12207, @aanm)
+* contrib/backporting: remove requires-janitor-review label (Backport PR #12341, Upstream PR #11986, @aanm)
+* docs: point cilium docs into a stable version of sphinx theme (Backport PR #12040, Upstream PR #12010, @genbit)
+* docs: re-design cilium docs theme (Backport PR #12040, Upstream PR #11803, @genbit)
+* envoy: Include detail in NACK warning (Backport PR #12341, Upstream PR #12016, @jrajahalme)
+* logo: change SVG file used for the logo (Backport PR #12040, Upstream PR #12002, @qmonnet)
+* Use right schema when validating CCNP in pre-flight upgrade step (Backport PR #12346, Upstream PR #12106, @aanm)
+
+**Other Changes:**
+* v1.6 backports 2020-06-10 (#12017, @jrajahalme)
```

Comment:
:thinking: seems like this one wasn't dereferenced properly by the release-notes script. Is it missing `upstream-prs` in the triple-backtick notes section of the issue description?

Reply:
No it wasn't that one. I'm not sure what this is.

Reply:
Ah but that one is missing the `upstream-prs` marker on the block. It just has a "raw" code block:

Reply:
Looks like this was one where the submitter manually wrote the triple-backtick section so it didn't have `upstream-prs` in the top. That causes release notes tool to ignore it.

Reply:
/cc @jrajahalme The `set-labels.py` quoted section needs to start with "```upstream-prs" to make sure the release notes script picks up the PRs correctly.

Reply:
Yep exactly